mastracode/src/web/.env.example

```
@@ -0,0 +1,114 @@
+MASTRACODE_TENANT_DB_ROOT=
+# optional — remote libSQL/Turso per tenant
+MASTRACODE_TENANT_DB_URL_TEMPLATE=
+# optional — separate remote vector DB per tenant
+MASTRACODE_TENANT_VECTOR_URL_TEMPLATE=
+# optional — auth token for the remote tenant storage DB
+MASTRACODE_TENANT_DB_AUTH_TOKEN=
+# optional — auth token for the remote tenant vector DB
+MASTRACODE_TENANT_VECTOR_AUTH_TOKEN=
+# optional — set to 1 to fail/warn at startup when no remote tenant DB template
+# is configured (local-file tenant DBs do not persist or share across replicas)
+MASTRACODE_REQUIRE_REMOTE_TENANT_DB=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MASTRACODE_REQUIRE_REMOTE_TENANT_DB key should go before the MASTRACODE_TENANT_DB_AUTH_TOKEN key

(UnorderedKey)